src/Testcontainers/Clients/TestcontainersConfigurationConverter.cs

```diff
@@ -25,7 +27,7 @@ public TestcontainersConfigurationConverter(ITestcontainersConfiguration configu
       this.ExposedPorts = new ToExposedPorts().Convert(configuration.ExposedPorts)?.ToDictionary(item => item.Key, item => item.Value);
       this.PortBindings = new ToPortBindings().Convert(configuration.PortBindings)?.ToDictionary(item => item.Key, item => item.Value);
       this.Mounts = new ToMounts().Convert(configuration.Mounts)?.ToList();
-      this.Networks = new ToNetworks().Convert(configuration.Networks)?.ToDictionary(item => item.Key, item => item.Value);
+      this.Networks = new ToNetworks().Convert(new[] { configuration })?.ToDictionary(item => item.Key, item => item.Value);
```

Comment:
What about:

```csharp
this.Networks = new ToNetworks().Convert(configuration.Networks)?.ToDictionary(
  network =>
  {
    return network.Key;
  },
  network =>
  {
    network.Value.Aliases = configuration.NetworkAliases.ToArray();
    return network.Value;
  });
```

Reply:
Code updated in [0a4072c](https://github.com/testcontainers/testcontainers-dotnet/pull/480/commits/0a4072cfc06a960bfc16130676a0a01e439e85cd)

Reply:
> I'm not happy with the converter in general 😀.

I used your suggestion since I see bigger refactoring can arraive :).